Libraries/LibJS/Runtime/StringPrototype.cpp

```diff
@@ -507,6 +520,88 @@ JS_DEFINE_NATIVE_FUNCTION(StringPrototype::slice)
     return js_string(vm, string_part);
 }
 
+JS_DEFINE_NATIVE_FUNCTION(StringPrototype::split)
+{
```

Comment:
If you don't want to implement the `@@split` part (step 2 in the spec), please leave a FIXME comment :)